front/package.json

```diff
@@ -93,41 +82,25 @@
     "pg": "^8.8.0",
     "pg-hstore": "^2.3.4",
     "pino": "^8.11.0",
-    "pino-pretty": "^10.0.0",
     "react": "^18.3.1",
-    "react-accessible-treeview": "^2.6.0",
     "react-beforeunload": "^2.5.3",
-    "react-confetti": "^6.1.0",
     "react-cookie": "^7.2.2",
-    "react-cropper": "^2.3.3",
     "react-dom": "^18.3.1",
     "react-dropzone": "^14.2.3",
     "react-hook-form": "^7.51.1",
-    "react-icons": "^4.9.0",
     "react-image-crop": "^10.1.8",
     "react-intersection-observer": "^9.13.1",
-    "react-katex": "^3.0.1",
     "react-markdown": "^8.0.7",
     "react-multi-select-component": "^4.3.4",
-    "react-p5": "^1.3.35",
-    "react-slick": "^0.29.0",
-    "react-syntax-highlighter": "^15.5.0",
     "react-textarea-autosize": "^8.4.0",
     "redis": "^4.6.8",
-    "rehype-katex": "^7.0.0",
```

Comment:
quite suspicious that this is really unused along with all remark/rehype plugins

Reply:
(some were moved to sparkle with the markdown component but there isn't a 1-1 mapping in particular the maths related ones)

Reply:
From what I can see, it's only used [here](https://github.com/dust-tt/dust/blob/20696ad30fa52ae7ce0a4d7506fbadfce7da68f9/sparkle/src/components/markdown/Markdown.tsx#L6)